cmd/podman/cp.go

```diff
@@ -68,11 +76,13 @@ func cpCmd(c *cliconfig.CpValues) error {
 	}
 	defer runtime.Shutdown(false)
 
-	extract := c.Flag("extract").Changed
-	return copyBetweenHostAndContainer(runtime, args[0], args[1], extract)
+	copyOpts := copyOptions{
+		extract: c.Extract,
```

Comment:
Why are we only filling one field in here?

If we're not using all of `copyOptions`, just pass in Extract to `copyBetweenHostAndContainer` and make a `copyOptions` in there